consumer/start-consumer.sh

```diff
@@ -0,0 +1,2 @@
+docker build -t my-kafka-consumer .
```

Comment:
could be better remove the `build` inside the start script or handle it appropriately

Reply:
Done, modified the script for both consumer and producer